sagemaker_neo_compilation_jobs/pytorch_vgg19_bn/pytorch-vgg19-bn.ipynb

```diff
@@ -120,7 +129,7 @@
     "pt_vgg = PyTorchModel(model_data=model_path,\n",
     "                      framework_version=framework_version,\n",
     "                      role=role,                               \n",
-    "                      entry_point='vgg19_bn.py',\n",
+    "                      entry_point='vgg19_bn_old.py',\n",
```

Comment:
Why is this using the old file?

Reply:
The notebook compares the performance of uncompiled VGG19 vs compiled VGG19 with PyTorch 1.2
I left the uncompiled model unchanged at PyTorch 1.2 and the inference container uses DLR. 
The compiled model was changed to PyTorch 1.4 and the inference container is updated.

With the update to the SDK however(https://github.com/aws/sagemaker-python-sdk/pull/1934/files#r498421790), even the uncompiled model will start using inference container with PyTorch 1.4. I can update the uncompiled flow preemptively to do this.

Reply:
Addressed